### PR TITLE
Added Routing

### PR DESCRIPTION
**Pull request by Meggie-H**

### CODE REVIEW

**Review by LesediMoko — DISMISSED** [review]
I see no clear issues with the routing setup, nice!

**Review by tanadendere — DISMISSED** [review]
The routing looks nicely setup to me!

File: src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+
+declare module '@tanstack/react-router' {
+  interface Router {
+    router: typeof router;
+  }
+}
 
 function App() {
   return (
     <>
```
Comment (Musa-Mabasa) [inline src/App.tsx]: A fragment is not needed in this case. It is only needed when you want to wrap multiple components or elements.

File: src/components/UserForm.tsx [hunk 2]
```diff
@@ -68,7 +70,7 @@ const UserForm = () => {
           type="submit"
           className="w-full rounded-md bg-blue-400 px-4 py-4 text-white hover:bg-blue-600 focus:outline-none focus:ring focus:ring-blue-300"
         >
-          Get your stats
+        <Link to={`/Dashboard`}>Get your stats</Link>
```
Comment (Musa-Mabasa) [inline src/components/UserForm.tsx:73]: Curly braces and backticks are not needed for sting literals. `to="/Dashboard"`

**Review by Musa-Mabasa — CHANGES_REQUESTED** [review]
Just some minor code refactoring, everything else looks good.

File: src/components/UserForm.tsx (hunk 2 above)
Comment (Meggie-H) [reply src/components/UserForm.tsx:73]: It does not work without it

File: src/App.tsx (hunk 1 above)
Comment (Meggie-H) [reply src/App.tsx]: Changed

**Review by Musa-Mabasa — APPROVED** [review]
The routing looks good.👍

**Review by tanadendere — APPROVED** [review]
The routing looks nicely setup to me!

**Pull request merged**